config/default/manager_default_images.yaml

```diff
@@ -25,6 +25,8 @@ spec:
           value: registry.redhat.io/rhosp-rhel8-tech-preview/osp-director-downloader:1.0
         - name: PROVISIONING_AGENT_IMAGE_URL_DEFAULT
           value: registry.redhat.io/rhosp-rhel8-tech-preview/osp-director-provisioner:1.0
+        - name: AGENT_IMAGE_URL_DEFAULT
+          value: quay.io/dprince/osp-director-operator-agent:0.0.1
```

Comment:
quay.io/dprince/osp-director-operator-agent should be quay.io/openstack-k8s-operators/osp-director-operator-agent

Reply:
agreed. Before I did that however I wanted to get feedback on the container name in general. Sounds like it is a go.